pkg/agent/route/interfaces.go

```diff
@@ -38,6 +38,12 @@ type Interface interface {
 	// It should do nothing if the routes don't exist, without error.
 	DeleteRoutes(podCIDR *net.IPNet) error
 
+	// AddServiceRoutes adds route for a Service IP.
+	AddServiceRoutes(svcIP net.IP, gwIP net.IP) error
```

Comment:
Could you add a few more comments here to explain this is for adding routes for ClusterIPs on Windows?

Reply:
Updated.